python/private/py_info.bzl

```diff
@@ -205,6 +220,7 @@ Python targets. These are accumulated from the transitive `deps`.
 The order of the depset is not guaranteed and may be changed in the future. It
 is recommended to use `default` order (the default).
 """,
+        "package": "TODO",
```

Comment:
Related discussion in Bazel Slack about asking if we should add this field: https://bazelbuild.slack.com/archives/CA306CEV6/p1743702775560119?thread_ts=1743702761.960199&cid=CA306CEV6